src/legacy/core_plugins/kibana/public/discover/index.html

```diff
@@ -1,58 +1,26 @@
 <discover-app class="app-container">
   <!-- Local nav. -->
-  <kbn-top-nav name="discover" config="topNavMenu">
-    <!-- Transcluded elements. -->
-    <div data-transclude-slots>
-      <!-- Breadcrumbs. -->
-      <div data-transclude-slot="topLeftCorner" class="kuiLocalBreadcrumbs">
-        <h1 id="kui_local_breadcrumb" class="kuiLocalBreadcrumb" ng-if="opts.savedSearch.id">
-          <span class="kuiLocalBreadcrumb__emphasis">
-            <button
-              class="kuiLink"
-              type="button"
-              id="reload_saved_search"
-              aria-label="{{::'kbn.discover.reloadSavedSearchAriaLabel' | i18n: {defaultMessage: 'Reload saved search'} }}"
-              tooltip="{{::'kbn.discover.reloadSavedSearchTooltip' | i18n: {defaultMessage: 'Reload saved search'} }}"
-              tooltip-placement="right"
-              tooltip-append-to-body="1"
-              ng-click="resetQuery()"
-            >
-              <span class="kuiIcon fa-undo small"></span>
-              {{::'kbn.discover.reloadSavedSearchButton' | i18n: {defaultMessage: 'Reload'} }}
-            </button>
-          </span>
-        </h1>
-        <div class="kuiLocalBreadcrumb">
-          <span data-test-subj="discoverQueryHits" class="kuiLocalBreadcrumb__emphasis">{{(hits || 0) | number:0}}</span>
-          <span
-            i18n-id="kbn.discover.hitsPluralTitle"
-            i18n-default-message="{hits, plural, one {hit} other {hits}}"
-            i18n-values="{ hits }"
-          ></span>
-        </div>
-      </div>
-
-      <!-- Search. -->
-      <div data-transclude-slot="bottomRow" class="fullWidth">
-        <search-bar
-          query="state.query"
-          screen-title="screenTitle"
-          on-query-submit="updateQueryAndFetch"
-          app-name="'discover'"
-          index-patterns="[indexPattern]"
-          filters="filters"
-          on-filters-updated="onFiltersUpdated"
-          show-date-picker="enableTimeRangeSelector"
-          date-range-from="time.from"
-          date-range-to="time.to"
-          is-refresh-paused="refreshInterval.pause"
-          refresh-interval="refreshInterval.value"
-          on-refresh-change="onRefreshChange"
-        ></search-bar>
-      </div>
-    </div>
-  </kbn-top-nav>
 
+  <kbn-top-nav2 
+    name="discover" 
+    config="topNavMenu"
+    show-search-bar="true"
+    show-date-picker="enableTimeRangeSelector"
+    query="state.query"
+    screen-title="screenTitle"
+    on-query-submit="updateQueryAndFetch"
+    app-name="'discover'"
+    index-patterns="[indexPattern]"
+    filters="filters"
+    on-filters-updated="onFiltersUpdated"
+    date-range-from="time.from"
+    date-range-to="time.to"
+    is-refresh-paused="refreshInterval.pause"
+    refresh-interval="refreshInterval.value"
+    on-refresh-change="onRefreshChange"
+  >
+  </kbn-top-nav2>
```

Comment:
I tested this in Discover, and noticed that the top nav items now stack on narrow screens, instead of the old implementation where they were inline. I'm guessing we want these inline? (Maybe a q for @snide)

### 7.0
![Screenshot 2019-07-15 17 14 18](https://user-images.githubusercontent.com/1608770/61255001-00777200-a724-11e9-9e4b-e63615a86b4a.png)

### This PR
![Screenshot 2019-07-15 17 14 59](https://user-images.githubusercontent.com/1608770/61255019-17b65f80-a724-11e9-8e74-e88bf72c7262.png)

Reply:
There's probably a wrapping EuiFlexGroup that needs `responsive=false`

Reply:
@cchaos @lukeelmers I actually think it's a great behavior, especially if we dare to think about mobile devices.

But I changed it for now.

Reply:
FWIW i felt that the other layout was more small-screen-friendly as it took up less vertical space & there are only a few buttons anyway. But I will defer to @cchaos on that :)

Reply:
I'll revert it to the old way it was, so we can proceed with this PR, and if we want, we can address this in a follow up PR